nuxeo-admin-console-web/angular-app/src/app/shared/services/network.service.ts

```diff
@@ -10,19 +13,34 @@ type EndpointName = keyof typeof REST_END_POINTS;
   providedIn: "root",
 })
 export class NetworkService {
-  constructor(private http: HttpClient,private nuxeoJsClientService: NuxeoJSClientService) {}
+  constructor(
+    private http: HttpClient,
+    private nuxeoJsClientService: NuxeoJSClientService
+  ) {}
 
   getAPIEndpoint = (name: EndpointName): string => {
     const config = REST_END_POINT_CONFIG[name];
     return `${this.nuxeoJsClientService.getApiUrl()}${config.endpoint}`;
   };
 
-  makeHttpRequest<T>(endpointName: EndpointName, data?: Record<string, unknown>): Observable<T> {
+  makeHttpRequest<T>(
+    endpointName: EndpointName,
+    data?: Record<string, unknown>
+  ): Observable<T> {
     const config = REST_END_POINT_CONFIG[endpointName];
     let url = this.getAPIEndpoint(endpointName);
     const method = config.method || "PUT";
     let params = new HttpParams();
 
+    if (data?.["urlParam"] && Object.keys(data?.["urlParam"]).length > 0) {
+      Object.entries(data?.["urlParam"]).forEach(([key, value]) => {
+        if (url.indexOf(key) > -1) {
+          url = url.replace(`{${key}}`, value);
```

Comment:
use replaceAll in case there are multiple areas in the url where the same param is to be replaced

Reply:
done